sdk/core/azure-core-test/inc/azure/core/test/test_base.hpp

```diff
@@ -406,9 +406,34 @@ namespace Azure { namespace Core { namespace Test {
      */
     void TearDown() override;
 
```

Comment:
THere are a bunchof whitespace differences here that are confusing the diff, not sure why they are there.

Reply:
i don't see those .